iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/iot/client/IoTConsensusClientPool.java

```diff
@@ -48,20 +49,25 @@ public SyncIoTConsensusServiceClientPoolFactory(IoTConsensusConfig config) {
     @Override
     public KeyedObjectPool<TEndPoint, SyncIoTConsensusServiceClient> createClientPool(
         ClientManager<TEndPoint, SyncIoTConsensusServiceClient> manager) {
-      return new GenericKeyedObjectPool<>(
-          new SyncIoTConsensusServiceClient.Factory(
-              manager,
-              new ThriftClientProperty.Builder()
-                  .setConnectionTimeoutMs(config.getRpc().getConnectionTimeoutInMs())
-                  .setRpcThriftCompressionEnabled(config.getRpc().isRpcThriftCompressionEnabled())
-                  .setPrintLogWhenEncounterException(
-                      config.getRpc().isPrintLogWhenThriftClientEncounterException())
-                  .build()),
-          new ClientPoolProperty.Builder<SyncIoTConsensusServiceClient>()
-              .setCoreClientNumForEachNode(config.getRpc().getCoreClientNumForEachNode())
-              .setMaxClientNumForEachNode(config.getRpc().getMaxClientNumForEachNode())
-              .build()
-              .getConfig());
+      GenericKeyedObjectPool<TEndPoint, SyncIoTConsensusServiceClient> clientPool =
+          new GenericKeyedObjectPool<>(
+              new SyncIoTConsensusServiceClient.Factory(
+                  manager,
+                  new ThriftClientProperty.Builder()
+                      .setConnectionTimeoutMs(config.getRpc().getConnectionTimeoutInMs())
+                      .setRpcThriftCompressionEnabled(
+                          config.getRpc().isRpcThriftCompressionEnabled())
+                      .setPrintLogWhenEncounterException(
+                          config.getRpc().isPrintLogWhenThriftClientEncounterException())
+                      .build()),
+              new ClientPoolProperty.Builder<SyncIoTConsensusServiceClient>()
+                  .setCoreClientNumForEachNode(config.getRpc().getCoreClientNumForEachNode())
+                  .setMaxClientNumForEachNode(config.getRpc().getMaxClientNumForEachNode())
+                  .build()
+                  .getConfig());
+      ClientManagerMetrics.getInstance()
```

Comment:
can we remove these magic strings using this way?
<img width="881" alt="image" src="https://github.com/apache/iotdb/assets/32640567/5311af3c-fb66-4754-b153-43c65ae55337">

Reply:
ok